autotest/t065_test_gridintersect.py

```diff
@@ -1,5 +1,6 @@
-import os
+import sys
 
+sys.path.insert(1, "..")
```

Comment:
Why does the flopy setup directory need to be inserted into the system path?

Reply:
I can remove this when merging these changes into my own PR. This test script's name is changed (no more `t***`) and the new test functions will need a couple other changes anyways (using `pytest.importorskip` for optional imports, etc)